src/controllers/userController.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from "express";
+import axios from "axios";
+import asyncRetry from "async-retry";
+import { User } from "../types/User";
+
+const transformUserData = (user: any): User => {
+  return {
+    id: user.id,
+    prefix: "",
+    firstName: user.name.split(" ")[0],
+    lastName: user.name.split(" ")[1],
+    email: user.email,
+    address: `${user.address.street} ${user.address.suite} ${user.address.city} ${user.address.zipcode}`,
+    geolocation: `(${user.address.geo.lat}, ${user.address.geo.lng})`,
+    companyName: user.company.name,
+  };
+};
+
+export const getUsers = async (req: Request, res: Response) => {
+  try {
+    const userData = await asyncRetry(async () => {
+      const response = await axios.get(
```

Comment:
Export this to a file called `UserService` where all of the axios remote calls will be done to get ready for the next steps.